index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Pixel's Photography</title>
+
+  <link rel="stylesheet" href="./stylesheets/index.css">
+  <script src="https://kit.fontawesome.com/8d20eebc7a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <a href="./index.html" class="logo">
+      <img src="./assets/logo1.png" alt="logo">
+    </a>
+
+    <!-- ============ Navigation ===================== -->
+    <nav>
+      <div class="nav-icon" onclick="toggleMenu()">
+        <i class="fa fa-bars fa-lg" aria-hidden="true"></i>
+      </div>
+
+      <div class="nav-links-container">
+        <ul class="nav-links">
+          <li class="hover"><a href="./about.html">About</a></li>
+          <li class="hover"><a href="#programs">Program</a></li>
+          <li class="hover"><a href="#featured">Instructors</a></li>
+          <li class="hover"><a href="#sponsor">Sponsor</a></li>
+          <li class="hover"><a href="#">News</a></li>
+          <li class="hover" id="campaign"><a href="#contact"><button type="button">Photo Campaign</button></a></li>
+        </ul>
+      </div>
+
+    </nav>
+  </header>
+
+
+  <main>
+
+    <!-- ============= Hero Section ================ -->
+    <section class="heroContainer">
+      <div class="overlay"></div>
+      <div class="heroContent heroText">
+        <h1>"Capture your Creativity!"</h1>
+        <p>Unlock Your Photography <br> Potential!</p>
+      </div>
+
+      <div class="heroContent heroIntro">
+        <p>Unleash your creativity with us through photography. Capture moments. <br>Embark on a transformative journey.</p>
+      </div>
+
+      <div class="heroContent dateLocation gray">
+        <h1>2023.06.12 (THU) ~ 20 (FRI)</h1>
+        <p>@Photography Cafe, Accra Street, Abuja</p>
+      </div>
+      <div id="programs"></div>
+    </section>
+
+    <!-- ============= Program Section ================ -->
+   <section class="programContainer">
+      <div class="programOverlay"></div>
+      <div>
+        <h1>Main Program</h1>
+        <hr>
+      </div>
+      
+      <div class="cardGrid">
+        <div class="card">
+          <div class="card-icon">
+            <i class="fas fa-camera"></i>
+          </div>
+          <div class="card-icon">
+            <h3 class="card-heading">Photography Courses</h3>
+          </div>
+          <p class="card-description">Explore a wide range of photography courses tailored for beginners</p>
+        </div>
+
+        <div class="card">
+          <div class="card-icon">
+            <i class="fas fa-graduation-cap"></i>
+          </div>
+          <div class="card-icon">
+            <h3 class="card-heading">Certified Instructors</h3>
```

Comment:
Good job implementing your  conference web page , however there is something you need to work on.
You will agree with me that these cards has the `hover` effect , meaning , when the user's mouse is on one card, it should unleash the `hover` effect as showed in the design provided by Cindy ,Kindly improve that 😄 

Your design 

![Capture](https://github.com/DevaWinner/capstone/assets/80458199/7ea13edf-a3a6-4cab-b6c9-cf167d497223)


Cindy design 

![Capture1](https://github.com/DevaWinner/capstone/assets/80458199/ac4742be-93bc-475a-9be9-c08d04e78f4a)

Reply:
Thank you for noting this. I have implemented it.